packages/superset-ui-chart-controls/src/components/ControlHeader.tsx

```diff
@@ -1,3 +1,4 @@
+/** @jsx jsx */
```

Comment:
How does that work? What's the benefit of using `/** @jsx jsx */` instead of importing React?